src/Chainweb/BlockHeader.hs

```diff
@@ -65,7 +33,7 @@ module Chainweb.BlockHeader
 , powTarget
 
 -- * BlockHeader
-, BlockHeader(..)
+, BlockHeader
```

Comment:
Would it make sense to restrict the lenses into `BlockHeader` that are exported from this module to be just getters?

Reply:
Done